DFC.App.FindACourse/DFC.App.FindACourse.csproj

```diff
@@ -12,15 +12,18 @@
 
```

Comment:
Should we be targeting .Net core 3.1 as this is a new project

Reply:
Upgraded to 3.1. Bloody pain.